pkg/exec/exec.go

```diff
@@ -96,3 +96,16 @@ func (p ProcessExecutor) RunProcess(executable string, execArgs ...interface{})
 
 	return nil
 }
+
+func (p ProcessExecutor) RunLongRunningProcess(executable string, execArgs ...interface{}) (*exec.Cmd, error) {
```

Comment:
I don't like the name of the method. `Run` is not correct because it doesn't start the process. Also, it doesn't have to be long-running.

Why not just call it `CreateProcess` and even use it from the other methods.

Reply:
makes sense. I am very bad at naming things. my daughter is called `girl` 😄